src/app/js/main.js

```diff
@@ -180,38 +180,47 @@ function submit_slide(slide) {
   text = window.DATA.log.join('\n')
   element.style = 'font-size: 22px'
   element.innerHTML = '<a href="data:text/plain;charset=utf-8,' +
-    encodeURIComponent(text) + '" download="Cytoscape_Testing_results.txt">Download testing results</a>' +
-    '<br/> and <br/>' +
-    '<button type="submit" id="jiraBtn" onclick="submitReport()">Submit Jira Report</button>'
+    encodeURIComponent(text) + '" download="Cytoscape_Testing_results.txt">Download Test Results</a>' +
+    '<br/> <br/>' +
+    '<button type="submit" id="jiraBtn" class="btn btn-primary" onclick="sendData()">Submit Jira Report</button>'
 
   slide.appendChild(element)
 }
 
-// call our server rest api 
-function submitReport() {
+// Process report submission to the server
+function sendData() {
+  //TODO: update with new logger once available
+  var tester = document.getElementById('name').value
+  var testFeedback = document.getElementById('feedback').value
+  var testEnv = JSON.stringify(window.DATA['responses'].init.user_environment);
+  log(testFeedback, 'User Feedback')
   text = window.DATA.log.join('\n')
+
   var strconfirm = confirm("Are you sure you want to submit the report?");
   if (strconfirm == true) {
-    var userFeedback = document.getElementById('feedback').value
-    addResponse('user_feedback', { 'feedback': userFeedback })
-    var tester = document.getElementById('name').value
     session.userName = tester
-    var env = JSON.stringify(window.DATA['responses'].init.user_environment)
-    var req_url = '/api/SubmitJira?env=' + env + '&tester=' + tester + '&fileData=' + text
-
-    fetch(req_url, { method: 'GET' })
-      .then((resp) => resp.json())
-      .then(function (data) {
-        log('Jira report submission request sent to api server');
-        let id = data.key;
-        log('Jira issue id is: ' + id)
-      })
-      .catch(function (error) {
-        log(error);
-      });
+    var url = '/receiveData';
+    var data = {
+      testerName: tester,
+      testerEnv: testEnv,
+      testerFeedback: testFeedback,
+      fileData: text
+    };
+
+    fetch(url, {
```

Comment:
This should be a post, not a fetch.

Reply:
It is a post Kemari. you're looking at the code that was removed:
fetch(url, {
      method: 'POST',
      body: JSON.stringify(data),
      headers: {
        'Content-Type': 'application/json'
      }
    }).then(res => res.json())
      .then(response => alert("Report Submitted Successfully. Jira ID: " + response.key))
      
      .catch(error => alert('Failed to Upload the test report to Jira:' + error));
  }